diff -ur build/externals/Source/netgen_project/cmake/cmake_modules/FindOpenCasCade.cmake build/externals/Source/netgen_project.1/cmake/cmake_modules/FindOpenCasCade.cmake
--- build/externals/Source/netgen_project/cmake/cmake_modules/FindOpenCasCade.cmake	2018-11-23 10:52:27.000000000 +0100
+++ build/externals/Source/netgen_project.1/cmake/cmake_modules/FindOpenCasCade.cmake	2019-05-26 10:30:29.036866000 +0200
@@ -6,48 +6,7 @@
 # OCC_LIBRARY_DIR    - where the OCC library directory can be found
 # OCC_LIBRARIES      - Link this to use OCC
 
-if(WIN32)
-    find_path(OCC_INCLUDE_DIR Standard_Version.hxx PATH_SUFFIXES inc ../inc)
-    find_library(OCC_LIBRARY TKernel)
-else(WIN32)
-    find_path(OCC_INCLUDE_DIR Standard_Version.hxx
-      /usr/include/opencascade
-      /usr/local/include/opencascade
-      /usr/include/oce
-      /usr/local/include/oce
-      /opt/opencascade/include
-      /opt/opencascade/inc
-    )
-    find_library(OCC_LIBRARY TKernel
-      /usr/lib
-      /usr/local/lib
-      /opt/opencascade/lib
-    )
-endif(WIN32)
-
-if(OCC_LIBRARY)
-    get_filename_component(OCC_LIBRARY_DIR ${OCC_LIBRARY} PATH)
-endif(OCC_LIBRARY)
-
-if(OCC_INCLUDE_DIR)
-    file(STRINGS ${OCC_INCLUDE_DIR}/Standard_Version.hxx OCC_MAJOR
-      REGEX "#define OCC_VERSION_MAJOR.*"
-    )
-    string(REGEX MATCH "[0-9]+" OCC_MAJOR ${OCC_MAJOR})
-    file(STRINGS ${OCC_INCLUDE_DIR}/Standard_Version.hxx OCC_MINOR
-      REGEX "#define OCC_VERSION_MINOR.*"
-    )
-    string(REGEX MATCH "[0-9]+" OCC_MINOR ${OCC_MINOR})
-    file(STRINGS ${OCC_INCLUDE_DIR}/Standard_Version.hxx OCC_MAINT
-      REGEX "#define OCC_VERSION_MAINTENANCE.*"
-    )
-    string(REGEX MATCH "[0-9]+" OCC_MAINT ${OCC_MAINT})
-
-    set(OCC_VERSION_STRING "${OCC_MAJOR}.${OCC_MINOR}.${OCC_MAINT}")
-endif(OCC_INCLUDE_DIR)
-
-
-set(OCC_LIBRARY_NAMES
+set(OCE_FIND_COMPONENTS
     TKBO
     TKBool
     TKBRep
@@ -80,13 +39,14 @@
     TKXSBase
 )
 
-foreach( libname ${OCC_LIBRARY_NAMES} )
-    find_library( ${libname} ${libname} ${OCC_LIBRARY_DIR} )
-    set(OCC_LIBRARIES ${OCC_LIBRARIES} ${${libname}})
-endforeach()
+find_package(OCE REQUIRED)
 
-include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(OCC REQUIRED_VARS OCC_INCLUDE_DIR VERSION_VAR OCC_VERSION_STRING ${OCC_LIBRARIY_NAMES})
+set(OCC_FOUND "${OCE_FOUND}")
+set(OCC_INCLUDE_DIR "${OCE_INCLUDE_DIRS}")
+get_property(OCC_LIBRARY_DIR TARGET TKernel PROPERTY LOCATION)
+# set(OCC_LIBRARY_DIR "${OCE_FOUND}")
+set(OCC_LIBRARIES "${OCE_LIBRARIES}")
+set(OCC_VERSION_STRING "${OCE_VERSION}")
 
 if(OCC_FOUND)
     message(STATUS "-- Found OpenCASCADE version: ${OCC_VERSION_STRING}")
